internal/rpc/jsonrpc/methods.go

```diff
@@ -1413,6 +1416,42 @@ func (s *Server) getMasterPubkey(ctx context.Context, icmd interface{}) (interfa
 	return masterPubKey.String(), nil
 }
 
+// getPeerInfo responds to the getpeerinfo request.
+// It gets the network backend and views the data on remote peers when in spv mode
+func (s *Server) getPeerInfo(ctx context.Context, icmd interface{}) (interface{}, error) {
+	w, ok := s.walletLoader.LoadedWallet()
+	if !ok {
+		return nil, errUnloadedWallet
+	}
+	n, err := w.NetworkBackend()
+
```

Comment:
remove spurious newlines between error assignments and the error handling

Reply:
done